### PR TITLE
Fix/ui run level logs

### PR DESCRIPTION
**Pull request by ajainarayanan**
Adds run level logs tab to workflows + flows + overview (no apps & no data) view

### CODE REVIEW

File: cdap-ui/app/features/mapreduce/mapreduce.less [hunk 1]
```diff
@@ -27,4 +27,9 @@ body.state-mapreduce{
       border-right: 1px solid black;
     }
   }
+  .btn.btn-default.no-border{
+    &:focus{
+      outline: none;
+    }
+  }
```
Comment (elfenheart) [inline cdap-ui/app/features/mapreduce/mapreduce.less]: Since this is repeated several times, should we extract this to either theme.less or common.less?

Comment (ajainarayanan) [inline cdap-ui/app/features/mapreduce/mapreduce.less]: Thought this is only specific for program types these should be in respective programs. Was not too sure how to create a program level styling.

File: cdap-ui/app/features/overview/templates/data-apps.html [hunk 2]
```diff
@@ -119,7 +122,10 @@ <h3 class="text-uppercase"> Apps </h3>
             data-button-icon="fa-plus"
             on-file-select="onFileSelected($files)">
         </my-file-select>
-        <a ui-sref="etlapps.create" class="btn btn-default">Add ETL App</a>
+        <a ui-sref="etlapps.create" class="btn btn-default">
+          <span class="fa fa-plus"></span>
+          <span>Add ETL App</span>
```
Comment (elfenheart) [inline cdap-ui/app/features/overview/templates/data-apps.html]: Change the wording to Add ETL Template?

**Comment by elfenheart** [comment]
Other than the comment, :+1: LGTM

**Pull request merged**